source/fundamentals/monitoring/cluster-monitoring.txt

```diff
@@ -0,0 +1,403 @@
+.. _golang-cluster-monitoring:
+
+==================
+Cluster Monitoring
+==================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, server, topology
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecols
+
+Overview
+--------
+
+This guide shows you how to use the {+driver-short+} to monitor topology
+events in a MongoDB instance, replica set, or sharded cluster. The
+driver creates topology events, also known as Server Discovery and
+Monitoring (SDAM) events, when there are any changes in the state of the
+instance or cluster that you are connected to. 
+
+You can use information about topology changes in your
+application, or you can monitor cluster changes to learn more about
+how they affect your application.
```

Comment:
You use this phrase on all three pages. How would you apply this information in your application? Could you make decisions about when to submit async requests? Could it influence batch sizes? Are there trends to look out for?

This is probably a tech review question, but without more information, this question doesn't really add anything.

Reply:
My guess is that there are a lot of reasons why someone could enable monitoring. I'll do some research and add some detail to each of the pages.